mypy_django_plugin/transformers/models.py

```diff
@@ -326,10 +326,8 @@ def run_with_model_cls(self, model_cls: Type[Model]) -> None:
                     related_manager_info = self.lookup_typeinfo_or_incomplete_defn_error(
                         fullnames.RELATED_MANAGER_CLASS
                     )  # noqa: E501
-                    # TODO: Use default manager instead of 'objects'
-                    # See: https://docs.djangoproject.com/en/dev/topics/db/queries/#using-a-custom-reverse-manager
-                    objects = related_model_info.get("objects")
-                    if not objects:
+                    default_manager = related_model_info.get("_default_manager")
+                    if not default_manager:
```

Comment:
Changed from `objects` to `_default_manager` with https://docs.djangoproject.com/en/dev/topics/db/queries/#using-a-custom-reverse-manager as justification ("By default the RelatedManager used for reverse relations is a subclass of the default manager for that model.")